.circleci/config.yml

```diff
@@ -64,7 +64,7 @@ workflows:
       - deploy:
           filters:
             tags:
-              only: /^v\d\.\d\.\d/
+              only: /v[0-9]+(\.[0-9]+)*/
```

Comment:
How about
v\d+(\.\d+){2} 

the problem I see is v1.4 is also valid as well v1

Reply:
Good idea, this will prevent to make mistakes with tags.